### PR TITLE
Jsonl addeed. 

### PR DESCRIPTION
**Pull request by 56616c6f72**
Cause James couldn't do it. 

Tested for both Hunt and Search by comparing hashes of outputs to expected output.

### CODE REVIEW

File: src/cli.rs [hunk 1]
```diff
@@ -727,6 +727,7 @@ pub fn print_json(
     rules: &BTreeMap<Uuid, Rule>,
     local: bool,
     timezone: Option<Tz>,
+    jsonl: bool,
```
Comment (alexkornitzer) [inline src/cli.rs:730]: Just nit picking but would call it something like `new_line`.

File: src/main.rs [hunk 2]
```diff
@@ -481,7 +488,9 @@ fn run() -> Result<()> {
                 if output.is_some() {
                     cs_eprintln!("[+] Writing results to output file...");
                 }
-                cli::print_json(&detections, hunter.hunts(), hunter.rules(), local, timezone)?;
+                cli::print_json(&detections, hunter.hunts(), hunter.rules(), local, timezone, false)?;
+            } else if jsonl {
```
Comment (alexkornitzer) [inline src/main.rs:492]: Could write this as:
```
} else if json || jsonl {
  cli::print_json(&detections, hunter.hunts(), hunter.rules(), local, timezone, jsonl)?;
}
```

File: src/cli.rs (hunk 1 above)
Comment (56616c6f72) [reply src/cli.rs:730]: I disagree. You are wrong

File: src/main.rs (hunk 2 above)
Comment (56616c6f72) [reply src/main.rs:492]: Smart. I do this.

**Pull request merged**